subprojects/libft/inc/ft_getline.h

```diff
@@ -16,8 +16,9 @@
 
 typedef struct	s_fd_state_store
 {
-	int			fd;
-	char		*fd_state;
+	int						fd;
+	char					*fd_state;
+	struct s_fd_state_store	*next;
 }				t_fd_state_store;
```

Comment:
i have no idea what this does and i don't like it,

why are you not using the t_list_conn?

Reply:
Rewriting the internal logic of `ft_getline` would have required a lot more work. And since `ft_getline` only used `ft_lstnew` I thought I would just change it so that it didn't need any list functions anymore.

I could change it to use `t_list_conn` but I don't really see the benefit of using it in `ft_getline`. All the state is internal anyway.

Reply:
fair enough, it's kinda a special case